gp-includes/template-links.php

```diff
@@ -364,18 +364,33 @@ function gp_link_glossary_delete( $glossary, $set, $text = '', $attrs = array()
  * @param WP_User $user A WP_User user object.
  */
 function gp_link_user( $user ) {
+	// This content is scaped in the gp_link_user_get() method.
```

Comment:
// This content is ~scaped~ escaped in the gp_link_user_get() method.
@amieiro here is a typo :)

Reply:
Thank you! Solved.